cmd/cat/cat.go

```diff
@@ -72,7 +72,12 @@ warc cat -n4 -P file1.warc.gz | feh -`,
 				c.offset = 0
 			}
 
-			c.filter = filter.NewFromViper()
+			c.filter = filter.New(
+				filter.WithMimeType(viper.GetStringSlice(flag.MimeType)),
+				filter.WithResponseCode(viper.GetString(flag.ResponseCode)),
+				filter.WithRecordIds(viper.GetStringSlice(flag.RecordId)),
+				filter.WithRecordTypes(viper.GetStringSlice(flag.RecordType)),
+			)
```

Comment:
This code is identical to two other changes in this PR, could this be simplified further?

Reply:
Discussed it IRL: we'll improve this in a follow up PR